examples/polymer-cli/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "polymer-cli",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "webpack-dev-server",
+    "storybook": "start-storybook -p 9001 -c .storybook"
+  },
+  "dependencies": {
+    "@polymer/polymer": "^2.2.0",
+    "@storybook/addon-actions": "3.3.0-alpha.3",
```

Comment:
Please change it to `^3.3.0-alpha.4` here and below

Reply:
@Hypnosphi Changing the version numbers of the dependencies causes an `Accessing nonexistent addons channel` error after bootstrapping again. Are there any recent changes in the addons that require other parts of the example code to change?

Reply:
The dependencies in `app/polymer should probably be updated as well

Reply:
Fixed with @ndelangen's merge and version changes